core/server/lib/request.js

```diff
@@ -13,14 +12,12 @@ module.exports = function request(url, options) {
         }));
     }
 
-    return new Promise(function (resolve, reject) {
-        return got(
-            url,
-            options
-        ).then(function (response) {
-            return resolve(response);
-        }).catch(function (err) {
-            return reject(err);
-        });
+    return got(
+        url,
+        options
+    ).then(function (response) {
```

Comment:
If there is no custom handling on `then` or `catch`, then there is no reason to add them here 🙃